src/pages/workspaces/workspace/Dashboard/DatasetAttributes.ts

```diff
@@ -0,0 +1,76 @@
+import { TooltipTrigger } from '@terra-ui-packages/components';
+import { cond } from '@terra-ui-packages/core-utils';
+import _ from 'lodash/fp';
+import { Fragment, ReactNode } from 'react';
+import { div, h, span } from 'react-hyperscript-helpers';
+import { SimpleTable } from 'src/components/table';
+import * as Style from 'src/libs/style';
+import { append } from 'src/libs/utils';
+import { WorkspaceInfo } from 'src/libs/workspace-utils';
+import { displayConsentCodes, displayLibraryAttributes } from 'src/pages/workspaces/workspace/library-attributes';
+
+interface DatasetAttributesProps {
+  attributes: WorkspaceInfo['attributes'];
+}
+
+export const DatasetAttributes = (props: DatasetAttributesProps): ReactNode => {
```

Comment:
Do you have a test workspace with dataset attributes?

Reply:
No: that was part of the reason I limited the changes to this part.

Reply:
FWIW it should be doable to create one for testing by calling the rawls [attribute API](https://rawls.dsde-dev.broadinstitute.org/#/workspaces/update_workspace)